sda-commons-server-kafka/src/test/resources/ssl/create-ca.txt

```diff
@@ -0,0 +1,20 @@
+# create CA
```

Comment:
Could this be an `.sh` file?

Reply:
Maybe also a short comment that this file was used to generate the example files and is here for documentation?

Reply:
👍